File: backend/Candle.py
class Candle:
    def __init__(self, open: float, close: float, wick: float):
        self.open = open
        self.close = close
        self.wick = wick

    def get_open(self) -> float:
        return self.open
    
    def get_close(self) -> float:
        return self.close
    
    def get_wick(self) -> float:
        return self.wick

File: backend/Candle.java
/*
 * The open price of a candle will be equal to the previous candle's close.
 * The prices will be determined by fantasy points
*/
package backend;

public class Candle {
    private double open;
    private double close;

    public double getOpen(Candle candle) {
        return candle.open;
    }

    public double getClose(Candle candle) {
        return candle.close;
    }
}
